app/views/users/new.html.erb

```diff
@@ -1,3 +1,18 @@
 <% provide :title, t("sign_up") %>
 <h1><%= t "sign_up" %></h1>
-<p><%= t "this_will" %><%= t "dot" %></p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user, url: signup_path) do |f| %>
```

Comment:
xóa ()

Form for rồi mà còn dùng url à ngoài form for còn có loại form khác không